README.md

```diff
@@ -1,2 +1,83 @@
-# lockstep-sdk-python
-Lockstep Platform API software development kit for Python
+# Fetch Invoice Sample Python Project
+
+Many types of products examine invoices for a customer and provide feedback on them. A typical product might analyze incoming invoices and add metadata like a **credit score** for each invoice. This tutorial explains how to iterate through invoices and examine them. 
+
+We use the [Query Invoices API](https://developer.lockstep.io/reference/get_api-v1-invoices-query) to retrieve a collection of invoices. To fetch a large number of invoices, we must use [filtering and pagination](https://developer.lockstep.io/docs/querying-with-searchlight). Here's how it works
+
+## Step 1: Declaring and initializing LockstepApi
+First,  you'll need to install LockstepAPI. You can do so in the terminal with 
+```bash
+pip install lockstep-sdk
+
+```
+Then you'll be able to start on your python file and utilize the lockstep API 
+```python
+import LockstepApi
+
+# env can also be prd or your own url addres
+env = 'sbx' 
+client = LockstepApi(env)
+client.with_api_key({INSERT_API_KEY})
+```
+
+**TODO: Add status result to verify API is connected**
```

Comment:
I think this would be just `client.status.ping()`, right?  Maybe with printing out the results